x-pack/test/saved_object_api_integration/common/suites/bulk_create.ts

```diff
@@ -219,8 +219,8 @@ export function bulkCreateTestSuiteFactory(context: FtrProviderContext) {
         });
 
         after(async () => {
+          await testDataLoader.deleteAllSavedObjectsFromKibanaIndex();
```

Comment:
Do we need this in bulk get as well?

Reply:
No, the `bulk_get` tests dont add any data that will conflict with other tests in the bulk_get suite